config/s3.js

```diff
@@ -0,0 +1,12 @@
+const AWS = require('aws-sdk');
+
+const s3Client = new AWS.S3({
+  region: 'ca-central-1',
+  credentials: {
+    accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+    secretAccessKey: process.env.AWS_SECRET_KEY,
+  },
+  computeChecksums: true,
+});
+
+module.exports = s3Client;
```

Comment:
It happens on occasion that the s3Client gets initialised with incorrect, or non-existent, environment variables. Unfortunately, this is not obvious until the user tries upload a profile picture and at that point the upload fails. The client's client connection should be validated after initialization to avoid errors down the line.